pilot/pkg/xds/bench_test.go

```diff
@@ -250,6 +254,43 @@ func BenchmarkSecretGeneration(b *testing.B) {
 	runBenchmark(b, v3.SecretType, secretCases)
 }
 
+var wdsCases = []ConfigInput{
+	{
+		Name:             "serviceentry-workloadentry",
+		Services:         100,
+		Instances:        1000,
+		KubernetesClient: true,
+	},
+}
+
+func BenchmarkAddressFullGeneration(b *testing.B) {
+	runBenchmark(b, v3.AddressType, wdsCases)
+}
+
+func TestAddressFullGeneration(t *testing.T) {
+	testBenchmark(t, v3.AddressType, wdsCases)
+}
+
+var wdsIncrementalCases = func() []ConfigInput {
+	cases := slices.Clone(wdsCases)
+	// Request a single resource
+	cases[0].PushRequest = &model.PushRequest{
```

Comment:
Can you clarify what this incremental case is describing?

Reply:
Yeah this is simulating a push of a single Workload. So like the case a single Pod object changes